modules/recommended-etcd-practices.adoc

```diff
@@ -6,17 +6,13 @@
 [id="recommended-etcd-practices_{context}"]
 = Recommended etcd practices
 
-For large and dense clusters, etcd can suffer from poor performance if the keyspace grows too large and exceeds the space quota. Periodically maintain and defragment etcd to free up space in the data store. Monitor Prometheus for etcd metrics and defragment it when required; otherwise, etcd can raise a cluster-wide alarm that puts the cluster into a maintenance mode that accepts only key reads and deletes.
+Because etcd writes data to disk and persists proposals on disk, its performance depends on disk performance.
+Etcd is not a particularly I/O intensive component, however it requires a low latency block device for an optimal performance and stability. Since etcd’s consensus protocol depends on persistently storing metadata to a log (WAL), etcd is very sensitive to disk write latency. Slow disks and disk activity from other processes can cause long fsync latencies.
```

Comment:
Suggested rewrite for style and to avoid capitalizing "etcd":
`Although etcd is not particularly I/O intensive, it requires a low latency block device for optimal performance and stability. Because etcd's consensus protocol depends on persistently storing metadata to a log (WAL), etcd is sensitive to disk-write latency.`